knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -595,7 +595,7 @@ func configureEventingKafka(spec serverlessoperatorv1alpha1.KnativeKafkaSpec) mf
 		if u.GetKind() == "Deployment" && u.GetName() == "kafka-controller" {
 
 			var disabledKafkaControllers = common.StringMap{
-				brokerController:  "broker-controller,trigger-controller",
+				brokerController:  "broker-controller,trigger-controller,namespaced-broker-controller,namespaced-trigger-controller",
```

Comment:
if you enable broker - you than will also have the namespaced ones.

Not sure if that actually reflects the subject of the PR

Reply:
Updated the PR title